app/assets/javascripts/client/inventories/InviteInventoryUserCtrl.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('InviteInventoryUserCtrl', InviteInventoryUserCtrl);
+
+  InviteInventoryUserCtrl.$inject = ['$scope', 'InventoryInvitation'];
+  function InviteInventoryUserCtrl($scope, InventoryInvitation) {
+    var vm = this;
+    vm.sendInvitation = function(invitation) {
+      $scope.alerts  = [];
+      $scope.addAlert = function(message) {
+        $scope.alerts.push({type: 'danger', msg: message});
+      };
+
+      $scope.closeAlert = function(index) {
+        $scope.alerts.splice(index, 1);
+      };
+      InventoryInvitation.create({ inventory_id: $scope.inventoryId }, invitation).$promise.then(function(response) {
+        var errors = response.data.errors;
```

Comment:
Two things:

 - We're in the context of a fulfilled promise; why would we expect errors?
 - We're not using the variable for anything.